scripting/get5/eventlogger.sp

```diff
@@ -78,10 +84,8 @@ public void EventLogger_SeriesStart() {
 
 public void EventLogger_MapVetoed(MatchTeam team, const char[] map) {
   EventLogger_StartEvent();
-
```

Comment:
Try not to change code in places you are not doing anything important. And maybe follow the existant style.

Like, keep the same style with blank lines in the event logger functions for `Pause` and `Unpause` too. 😊